fair_mango/dataset/dataset.py

```diff
@@ -127,6 +145,19 @@ def plot_groups(
         figsize: tuple[int, int] = (16, 6),
         dpi: int = 200,
     ):
```

Comment:
suggestion: add return type `None`

Reply:
This function is removed in another refactoring [branch](https://github.com/datategy/Fair-Mango/tree/refactor/reduce-complexity) (still did not create a PR for it). It will be introduced again along with a more capable and complete visualisation module in the future.